src/mockServiceWorker.js

```diff
@@ -316,7 +320,11 @@ function sendToClient(client, message) {
       resolve(event.data)
     }
 
-    client.postMessage(message, [channel.port2])
+    const transfer = [channel.port2]
+    if (body) {
+      transfer.push(body)
```

Comment:
Can array buffer be transmitted through a message channel without pushing it to the transport?

Reply:
it is converted to an opaque object if not transferred.